recipe/meta.yaml

```diff
@@ -12,10 +12,10 @@ source:
   patches:
     - patches/pkgconfig_generate_windows_llvm.patch  # [win]
 
-{%- set number = 7 %}
-{%- if license_family == 'gpl' %}
-{%- set number = number + 100 %}
-{%- endif %}
+{% set number = 8 %}
+{% if license_family == 'gpl' %}
+    {% set number = number + 100 %}
+{% endif %}
```

Comment:
@hmaarrfk 
Not sure if there's a jinja syntax detail I don't know (I'm actually surprised this worked so far), but just double-checking that removing the hyphen here is OK. It seems to break the bot's ability to bump the build number, at least.

Reply:
without the `-`, an empty line is added in the rendered file.

I guess empty lines don't matter, but it keeps the final `meta.yaml` file that is included in the eventual package cleaner.

Good to go from me. It would be nice to have time to submit a bug report upstream to the bot.